src/analysis/retail/warlock/affliction/constants.ts

```diff
@@ -32,8 +32,8 @@ export const getDotDurations = (combatant: Combatant): Record<number, number> =>
     : defaultDurations;
 
 // Shadow Embrace modifiers for Drain Soul / No Drain Soul
-export const shadowDefaultEmbraceModifier = 0.04;
-export const shadowDrainSoulEmbraceModifier = 0.02;
+export const Shadow_Default_Embrace_Modifier = 0.04;
+export const Shadow_Drain_Soul_Embrace_Modifier = 0.02;
```

Comment:
All the letters here should be uppercase. eg `SHADOW_DEFAULT_EMBRACE_MODIFIER`.

Reply:
Done